examples/stable_diffusion_xl/train.py

```diff
@@ -125,6 +125,7 @@ def get_parser_train():
         default=None,
         help="Max number of ckpts saved. If exceeds, delete the oldest one. Set None: keep all ckpts.",
     )
+    parser.add_argument("--resume_step", type=int, default=0, help="resume from step_n")
```

Comment:
I think we should record the step number in the saving checkpoint so that it can be loaded when resume training  instead of manual setting.

Reply:
> I think we should record the step number in the saving checkpoint so that it can be loaded when resume training instead of manual setting.

It makes sense that we can continue to optimize in the next PR @townwish4git